### PR TITLE
Case Management Styling

### PR DESCRIPTION
**Pull request by biyeun**
@millerdev @emord 
@dimagi/product 

screen shot 2015-10-09 at 10 56 32 pm
screen shot 2015-10-09 at 10 56 54 pm
screen shot 2015-10-09 at 10 57 00 pm
screen shot 2015-10-09 at 10 57 07 pm
screen shot 2015-10-09 at 10 57 15 pm
screen shot 2015-10-09 at 11 47 10 pm
screen shot 2015-10-09 at 11 47 16 pm

### CODE REVIEW

File: src/widgets.js [hunk 1]
```diff
@@ -257,16 +257,24 @@ define([
...
-                    template: '<div contenteditable="false" class="popover">' +
-                        '<div class="arrow"></div>' +
+                    xpath = match ? match[1] : datavalue,
+                    displayId = $this.clone().children().remove().end().text(),
+                    labelMug, labelText;
+                if (/^\/data\//.test(xpath)) {
+                    labelMug = mug.form.getMugByPath(xpath);
+                    labelText = labelMug ? labelMug.p.labelItext.get() : "";
+                } else {
+                    return;
```
Comment (millerdev) [inline src/widgets.js]: Discussed with @amsagoff and we decided to disable the popover for case properties until we have support for #case_type/property special syntax.

**Comment by millerdev** [comment]
@orangejenny would you mind reviewing my last two commits. Feel free to merge if you think they look good (or simply sign off and I'll merge). Once this is merged I'll push to prod and make a QA ticket.

**Comment by orangejenny** [comment]
looks good to me

**Comment by amsagoff** [comment]
HOORAYYYYY!!!

**Comment by millerdev** [comment]
This will go to prod with the next deployment.

**Pull request merged**